src/app/zap-templates/zcl/data-model/chip/tls-client-management-cluster.xml

```diff
@@ -49,36 +49,40 @@ Git: 1.4-819-g9894ad1b9
     <globalAttribute code="0xFFFD" side="either" value="1"/>
     <attribute code="0x0000" side="server" define="MAX_PROVISIONED" type="int8u">MaxProvisioned</attribute>
     <attribute code="0x0001" side="server" define="CURRENT_PROVISIONED" type="int8u">CurrentProvisioned</attribute>
-    <attribute code="0x0002" side="server" define="MAX_IN_USE" type="int8u">MaxInUse</attribute>
-    <attribute code="0x0003" side="server" define="CURRENT_IN_USE" type="int8u">CurrentInUse</attribute>
+    <attribute code="0x0002" side="server" define="MAX_IN_USE" type="int8u">CurrentInUse</attribute>
```

Comment:
Odd-looking `define` there.

Reply:
Thanks for catching that! I think alchemy borked because of the overlap in the names 😅